src/components/protection/AddProtectionSingle.vue

```diff
@@ -296,7 +296,8 @@ export default class AddProtectionSingle extends BaseComponent {
   }
 
   get whitelistWarning() {
-    const msg = i18n.t("pool_not_approved");
+    const msg =
```

Comment:
Are we meant to be losing the ```i18n.t``` feature?

Reply:
no, current on master branch, it use `i18n.t`. just following on master branch to merge